specs/example.spec

```diff
@@ -0,0 +1,31 @@
+# Specification Heading
+
```

Comment:
why you have not removed the unnecessary spec file?

Reply:
Done